forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/KeyAware.kt

```diff
@@ -54,6 +53,12 @@ class KeyAware(private var userId: String) {
         return KeyPair(publicKey, privateKey, key)
     }
 
+
+    fun getPrivateKey(keyAlias: String = key): PrivateKey? {
```

Comment:
What about this? Could this function be accessible to a bad actor?

Reply:
@cerebrl actually its a good catch, the reason is they cannot directly access this , but they can inherit this class and invoke this method, so i am going to remove that method and directly consume the private key without exposing this